uiowa/uiowa
# DDev followup: use `drupal:tests` in CI for functional JS tests and reconfigure/document local testing.

Related to #4538 

This PR removes the BLT configuration required for the ddev chromedriver container to work from our `blt.yml` file. The reason for this is so we don't have to then override those values to the defaults provided by BLT in `ci.blt.yml`. The other benefit is should those values change in BLT, we'll get those updated values.

This change means it is necessary to specify more values in your `blt/local.blt.yml` files if you'd like to run Drupal tests locally. I've updated `tests/README.md` to reflect this.

Additionally, this PR changes how our tests our run in CI. We now use the `tests:drupal` command to run all phpunit tests and configure phpunit to use our repo config. This ensures the necessary chrome and chromedriver requirements for functional JS test.

### Random
Added `ddev phpcs`, `ddev phpcbf` and `ddev phpunit` commands because I got sick of typing `./vendor/bin/...`.

### Testing
- Reconfigure your `blt/local.blt.yml` file according to the `tests/README.md` instructions to run Drupal core/contrib test.
- Run `blt tests:drupal` and make sure it works.
- Run `blt tests:phpunit` and make sure the uiowa_search functional JS test located at https://github.com/uiowa/uiowa/blob/main/docroot/modules/custom/uiowa_search/tests/src/FunctionalJavascript/SearchTest.php is not marked as skipped.
- Make sure the same test is not marked as skipped in CI log output. 
  - Previous example of skipped test: https://app.travis-ci.com/github/uiowa/uiowa/builds/243121493#L2168
  - Latest build (could change): https://app.travis-ci.com/github/uiowa/uiowa/builds/243128919#L2171
